jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationExplorerNodes.kt

```diff
@@ -54,18 +56,37 @@ class CloudFormationServiceNode(project: Project) : AwsExplorerServiceRootNode(p
     }
 }
 
-class CloudFormationStackNode(project: Project, val stackName: String, private val stackStatus: StackStatus, val stackId: String) :
-    AwsExplorerResourceNode<String>(project, CloudFormationClient.SERVICE_NAME, stackName, AwsIcons.Resources.CLOUDFORMATION_STACK),
+class CloudFormationStackNode(
+    project: Project,
+    val stackName: String,
+    private val stackStatus: StackStatus,
+    val stackId: String
```

Comment:
So, this is formatted by the official Kotlin formatter. Austin also commented about formatting which I replied above, pasting my reply again here:

I see there are only 2 predefined Kotlin code styles in preferences:

1. **Kotlin obsolete IntelliJ IDEA codestyle**: This is the default style I had in my IDE which uses continuation indent (sized 8) after parenthesis '(' which doesn't look very nice.

2. **Kotlin style guide**: Following https://kotlinlang.org/docs/reference/code-style-migration-guide.html, I changed code style in my IDE (Editor > Code Style > Kotlin > Set from... > Predefined Style > Kotlin style guide) and added 'kotlin.code.style=official' property to the gradle.properties file so that all of us use the same official code style which makes formatting consistent with the Kotlin Coding Conventions. This new style uses the indent size 4 after parenthesis. The continuation indent size is also 4 in this style, so there will be some formatting changes in the files I am touching.

**Note**: if we prefer using a different code style than these predefined ones, `.idea/codeStyle` folder should be committed so that all project developers could share the same code style.

Reply:
Reverted formatting changes.